src/main/kotlin/uk/gov/dluhc/printapi/dto/AddressDto.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.dluhc.printapi.dto
+
+data class AddressDto(
+    var street: String,
+    var postcode: String,
+    var `property`: String? = null,
```

Comment:
Probably don't need the backticks on property (I think it's related to DynamoDB elsewhere)